app/waf/src/controllers/wafpolicy.controller.ts

```diff
@@ -89,9 +89,12 @@ export class WafpolicyController extends BaseController {
   ): Promise<CollectionResponse> {
     return new CollectionResponse(
       Wafpolicy,
-      await this.wafpolicyRepository.find(filter, {
-        tenantId: await this.tenantId,
-      }),
+      await this.wafpolicyRepository.find(
+        {
```

Comment:
Add its test case?

Reply:
we need multiple tenants in context and change them dynamically.

Reply:
Why for that?

The test should be 
1. create the wafpolicy data whose property `public === true` and `tenantId === t1`.
2. access the api with `tenantId === t2` and check the return value is not empty and tenantId === t1. 

Is it?